Dockerfile

```diff
@@ -2,7 +2,7 @@
 # CREATE UPDATED BASE IMAGE
 # ========================================
 
-FROM mcr.microsoft.com/powershell:debian-bullseye-slim AS base
+FROM debian:bullseye-slim AS base
```

Comment:
MS now released image with arm support. Let' stry one of these: https://hub.docker.com/_/microsoft-powershell (search for arm)

Reply:
@avnes see above discussion